roles.txt

```diff
@@ -0,0 +1,13 @@
+roles/artifactregistry.admin
+roles/cloudsql.admin
+roles/compute.networkAdmin
+roles/iam.serviceAccountAdmin
+roles/iam.serviceAccountUser
```

Comment:
"User" role might be redundant if "Admin" is already present on Service Account, unless it needs serviceAccount.actAs permission. [Roles documentation](https://cloud.google.com/iam/docs/understanding-roles#iam.serviceAccountUse)

Reply:
actAs permission might be required for the SA to impersonate the cloud build SA.